backend/src/api/chat/chat.api.ts

```diff
@@ -0,0 +1,51 @@
+import { Router } from 'express';
+import { validateSchema } from '~/middlewares';
+import { message as validationMessageSchema } from '~/validation-schemas';
+import { ApiPath, HttpCode, ChatsApiPath } from '~/common/enums';
+import { chat as chatService } from '~/services/services';
+
+const initChatApi = (apiRouter: Router): Router => {
```

Comment:
```suggestion
const initMessagesApi = (apiRouter: Router): Router => {
```
Remove `chat` word from everywhere